src/main/java/org/isf/examination/gui/PatientExaminationEdit.java

```diff
@@ -1464,18 +1464,36 @@ public SwingActionSavePatientExamination() {
 
 		@Override
 		public void actionPerformed(ActionEvent actionEvent) {
-
-			try {
-				examinationBrowserManager.saveOrUpdate(patex);
-				modified = false;
-			} catch (OHServiceException ohServiceException) {
-				MessageDialog.showExceptions(ohServiceException);
+			double weight = patex.getPex_weight();
+			int height = patex.getPex_height();
+
+			if (weight != 0 && height != 0) {
+				if (weight == ExaminationParameters.WEIGHT_MIN || weight == ExaminationParameters.WEIGHT_MAX || height == ExaminationParameters.HEIGHT_MIN || height == ExaminationParameters.HEIGHT_MAX) {
+					int response = JOptionPane.showConfirmDialog(null, MessageBundle.getMessage("angal.patient.examination.defaultvalue.msg"), "Confirm", JOptionPane.YES_NO_OPTION);
```

Comment:
I think that this message should be shown when `weight == ExaminationParameters.WEIGHT_INIT && weight == ExaminationParameters.WEIGHT_INIT` and we should change it to: "`Some values are equal to the default values. Do you really want to confirm them?`" in order to reuse it also for others `_INIT` parameters.

While for MIN and MAX we should shown a message like "`Some entered values are equal to their minimum or maximum values. Do you really want to confirm them?`" in order to reuse it also for others `_MIN` and `_MAX` parameters.

Wdyt?

Reply:
Why aren't you using `MessageDIalog.yesNo()` instead of `showConfirmDIalog()`?   Then you wouldn't have to worry about the fact that you didn't localize the `Confirm` string and this interaction would match all the other dialogs in the project.